go/chat/helper.go

```diff
@@ -41,15 +41,15 @@ type sendHelper struct {
 	ident       keybase1.TLFIdentifyBehavior
 
 	canonicalName string
-	topicName     string
+	topicName     *string
 	tlfID         chat1.TLFID
 	convID        chat1.ConversationID
 	triple        chat1.ConversationIDTriple
 
 	globals.Contextified
 }
 
-func newSendHelper(g *globals.Context, name string, topicName string,
+func newSendHelper(g *globals.Context, name string, topicName *string,
```

Comment:
Maybe, but right now they don't, so I just didn't bother with it. If we ever wanted to send attachments in the same manner we send text with `SendTextByName` (a theoretical `SendAttachmentByName`), then we would probably integrate it then. I don't anticipate that will ever be a thing though.